### PR TITLE
Added myself to the contributor's list

### PR DESCRIPTION
**Pull request by Sidhved**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ Thanks goes to these Wonderful People. Contributions of any kind are welcome! :g
     <td align="center"><a href="https://github.com/supzi-del"><img src="https://avatars.githubusercontent.com/u/78655439?v=4" height="120px" width="120px"/><br/><sub><b>Supritha</b></sub></a></td>
     <td align="center"><a href="https://github.com/Vedant-Jayesh-Oza"><img src="https://avatars.githubusercontent.com/u/75005433?v=4" height="120px" width="120px"/><br/><sub><b>Vedant Jayesh Oza</b></sub></a></td>
     <td align="center"><a href="https://github.com/jigar-sable"><img src="https://avatars.githubusercontent.com/u/64949957?v=4" height="120px" width="120px"/><br/><sub><b>Jigar Sable</b></sub></a></td>
```
Comment (akshitadixit) [inline README.md:170]: Hey @Sidhved this be fine. I'll however suggest you to shift yours and @jigar-sable 's columns to the next row in the table for better visibility on both mobile and desktop.

**Review by akshitadixit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sidhved) [reply README.md:170]: Ohkk sure I'll get that resolved

**Comment by Sidhved** [comment]
Hey @akshitadixit 
I have made the necessary change
Could you please review once it and merge?

**Review by akshitadixit — APPROVED** [review]
Looks good to go!

**Pull request merged**